homeassistant/components/homekit/type_sensors.py

```diff
@@ -128,21 +153,244 @@ def __init__(self, *args):
         """Initialize a AirQualitySensor accessory object."""
         super().__init__(*args, category=CATEGORY_SENSOR)
 
-        serv_air_quality = self.add_preload_service(
-            SERV_AIR_QUALITY_SENSOR, [CHAR_AIR_PARTICULATE_DENSITY]
+        """determine the charciteristics of the sensor based on the curent attributes"""
```

Comment:
For single line comments you should use `# `. Triple quotes aren't comments, they are strings but just not assigned to variables. (The places where you can see triple quotes used as comments are [docstrings](https://www.python.org/dev/peps/pep-0257/)).

Reply:
aaaaah. haven't done much with python. thanks for the info. fixed.